spec/controllers/hq/users_controller_spec.rb

```diff
@@ -31,4 +31,33 @@ def foobar_action
     expect(response).to render_template 'index'
   end
 
+  describe '#edit and #update' do
```

Comment:
Again, the Partners controller and related specs have established a generally agreed-upon paradigm. Is there a particular reason for the differences here?